descriptors/ModuleDescriptor-template.json

```diff
@@ -1,29 +1,76 @@
 {
   "id": "${artifactId}-${version}",
-  "name": "The blueprint module that could be used as a template for folio spring based backend modules.",
+  "name": "Search Module",
   "provides": [
     {
-      "id": "spring-template",
-      "version": "1.0",
-      "handlers": []
+      "id": "indexes",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/search/index/indices",
+          "permissionsRequired": ["search.index.indexes.item.post"]
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/search/index/mappings",
+          "permissionsRequired": ["search.index.mappings.item.post"]
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/search/index/resources",
+          "permissionsRequired": ["search.index.resources.collection.post"]
+        }
+      ]
     },
     {
-      "id": "_tenant",
-      "version": "1.2",
-      "interfaceType": "system",
+      "id": "search",
+      "version": "0.1",
       "handlers": [
         {
-          "methods": ["POST"],
-          "pathPattern": "/_/tenant"
-        }, {
-          "methods": ["DELETE"],
-          "pathPattern": "/_/tenant"
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/search/instances",
+          "permissionsRequired": ["search.instances.collection.get"]
         }
       ]
     }
   ],
-  "permissionSets" : [],
-  "requires": [],
+  "requires": [
+    {
+      "id": "instance-storage",
+      "version": "7.5"
+    }
+  ],
+  "permissionSets" : [
+    {
+      "permissionName": "search.index.indexes.item.post",
+      "displayName": "Search - create an index",
+      "description": "Create an index"
```

Comment:
Either we change `create` to `creates`, or change Line 60-61, Line 65-66 and 70-71.

Reply:
Changed to `creates`, thanks!